opm/output/Wells.hpp

```diff
@@ -0,0 +1,173 @@
+/*
+  Copyright 2016 Statoil ASA.
+
+  This file is part of the Open Porous Media project (OPM).
+
+  OPM is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  OPM is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with OPM.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef OPM_OUTPUT_WELLS_HPP
+#define OPM_OUTPUT_WELLS_HPP
+
+#include <map>
+#include <stdexcept>
+#include <string>
+#include <vector>
+
+namespace Opm {
+
+    namespace data {
```

Comment:
Perhaps outputdata would be better? I think the the class names `Rates`, `Well` and `Wells` etc. are nice but very generic, and while they won't collide (since hardly anything else uses the data namespace) it would be better to use a more output-specific namespace. That way we can keep the nice names and be safe that we avoid any confusion.